src/tigris.ts

```diff
@@ -238,3 +237,52 @@ export class Collection<T extends TigrisCollectionType> {
  * Default instance of the Tigrisclient
  */
 export default new Tigris({serverUrl: `${process.env.TIGRIS_SERVER_URL}`});
+
+export type Filter<T extends string | number | boolean> = {
```

Comment:
I am assuming extending the filter is a non-breaking change?

Reply:
yes more types can be added here without breaking the existing usage.